oo/classical-inheritance-es5.md

```diff
@@ -0,0 +1,164 @@
+Before the introduction of classses in ES6 it was common to mimic class inheritance with constructor Functions and [Object.create()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create). Its also possible to use factory functions for this purpose. As these are not true classes we sometimes call these "pseudo-classes". The examples below show how to implement classical inheritance with the tools available in ES5:
```

Comment:
It makes sense to not make comparisons in the intro and I'll tweak this. 

It also makes sense to link to related concepts or different implementations of the same concept. Ie. Just because you can do inheritance with constructor functions doesn't mean its a good idea. You might use object composition (related concept) or the `class` implementation depending on your purpose.

Now, we don't really expect junior developers to be able to make a call here.   But there's something there about us taking a view on  best practices or presenting all these concepts from a neutral point of view, and helping them on the path to forming grounded but pragmatic opinions about all these different options. 


As a general noticing I find myself exploring a 'concepts web' - noticing how one concept links to another, as I write these. Partially  this has come from making a distinction between a concept and an implementation of that concepts (in this case `inheritance` and implementations with `class`,  constructor functions, and factory functions.)

cc @pietgeursen